.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
I'd say that you need to add this file to your `.gitignore`, but ask Raul first.

Reply:
Raúl say that this file will be excluded of the .gitignore. That help to others developer...  ;)